web/views/partials/header.hbs

```diff
@@ -0,0 +1,37 @@
+<header>
+      <nav class="nav" role="navigation">
+        <a href="/"><img class="nav__logo" src="/static/images/logo.png" alt="Sabka Bazaar Logo" />
+        </a>
+        <ul class="nav__bar">
+          <li class="nav__bar--link"><a href="/">Home</a></li>
+          <li class="nav__bar--link"><a href="/products">Products</a></li>
+        </ul>
+        <div class="nav__end">
+          <div class="nav__end-register-box">
+            <a class="nav__end-register-box--link" href="/SignIn"
+              >SignIn</a
+            >
+            <a class="nav__end-register-box--link" href="/SignUp"
+              >Register</a
+            >
+            
+          </div>
+          <div class="nav__end-cart">
+            <a href="/cart" id="myBtn"><img src="/static/images/cart.svg" class="nav__end-cart--logo"></a>
+            <div class="nav__end-cart--count"> {{cart.count}} items </div>
+          </div>
+        </div>
+        <div class="menu-toggle">
+          <input type="checkbox" />
+          <span></span>
+          <span></span>
+          <span></span>
+          <ul class="menu">

```

Comment:
Why are you having duplicate navigation? This is bad for SEO and Accessibility. Use the same navigation DOM  items to build two views for mobile and other devices.

Reply:
This NAV is still duplicate